packages/atclient/3rdparty/cJSON/README.md

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Delete white space before header line.

I would suggest linting all markdown with [pymarkdownlnt](https://pypi.org/project/pymarkdownlnt/), which we now have in automated tests for some of our docs repos.

Reply:
https://github.com/atsign-foundation/at_c/pull/91/commits/19951129e451f10b8539459c1e16f3132f796c0d